miso-web/src/main/webapp/scripts/pane.js

```diff
@@ -178,11 +184,16 @@ var Pane = (function() {
         content: contentDiv
       };
     },
-    updateTiles: function(div, transform, url, query) {
+    
+    // add specifies any special tiles to be added to the bottom of the pane
+    updateTiles: function(div, transform, url, query, add) {
+      if(add === undefined){
```

Comment:
I think `pane_instrument_status.js` needs to be updated to match the new signature of `updateTiles`. It should get `,null, []` added to the argument list rather than trying to deal with the undefined here.